### ISSUE TITLE
pop3 Error 

### ISSUE THREAD

**Issue by moonya0830** [jstedfast/MailKit#457]
Hi,

When I use pop3Client Connect to Exchange Server:

pop.Connect(strPop3Host, 110, true);

ProtocolLogger is Null.

And Exception Error is:

![image](https://cloud.githubusercontent.com/assets/25735284/22874435/f3797b46-f200-11e6-9809-ef974ed26866.png)

=====

When I use pop3Client Connect to Exchange Server:

pop.Connect(strPop3Host, 110, SecureSocketOptions.StartTls);

ProtocolLogger is:

![image](https://cloud.githubusercontent.com/assets/25735284/22874024/b128226c-f1fe-11e6-825d-a7fe57aac7d1.png)

And Exception Error is:

![image](https://cloud.githubusercontent.com/assets/25735284/22874596/e4d21a84-f201-11e6-8f10-da32a6c0a578.png)

=====

But I use Outlook is OK.

I don't understand, Please help me to debug this.

Thank you.

**Comment by jstedfast** [comment]
The exception in your first example is caused because port 110 does not support SSL-wrapped connections (e.g. SSL enabled from the very start).

The exception in your second example, which you set to use StartTLS (which is correct, btw), has a problem because the server's SSL certificate is invalid. Most likely your server is using a self-signed certificate (or else the certificate has expired?) and so you will need to implement your own certificate validation logic.

In other words, you'll have to provide your own `pop.ServerCertificateValidationCallback`.

A very simple example of how to do this would be:

```csharp
pop.ServerCertificateValidationCallback = (s,c,h,e) => true;
```

**Comment by moonya0830** [comment]
Thank you for your help. :)
